pkg/fixedpoint/slice.go

```diff
@@ -0,0 +1,24 @@
+package fixedpoint
+
+type Slice []Value
```

Comment:
move to `pkg/types/value_slice.go`?
[pkg/types/value_map.go](https://github.com/c9s/bbgo/blob/7225a597f205b151ad5dc6837bd55ff8a7d58435/pkg/types/value_map.go)